package.json

```diff
@@ -66,6 +67,7 @@
     "react-native": "0.42.x",
     "react-native-mock": "^0.3.1",
     "react-test-renderer": "~15.4.2",
+    "react-with-styles-interface-aphrodite": "^3.1.1",
```

Comment:
this seems like it might create some kind of recursive dep graph; i have no concrete concerns but this makes me think of scary noises

Reply:
Yeah I'm uneasy about this. I only really need the flush functionality I believe, so maybe I just port this stuff over? https://github.com/airbnb/react-with-styles-interface-aphrodite/blob/master/src/aphroditeInterfaceFactory.js#L22

Reply:
i mean, we could use `import { flushToStyleTag } from 'aphrodite/lib/inject';` directly, i suppose

Reply:
I'd like to call the interface though since that's what this line calls: https://github.com/airbnb/react-with-styles/blob/master/src/withStyles.jsx#L80

so just in case it changes in the future